functional/cypress-tests/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "plugins": [
+      "cypress"
+    ],
+    "rules": {
+        "cypress/no-assigning-return-values": "error",
+        "cypress/no-unnecessary-waiting": "error",
+        "cypress/assertion-before-screenshot": "warn",
+        "quotes": ["warn", "single"],
+        "newline-per-chained-call": ["warn", {"ignoreChainWithDepth": 1}],
+        "indent": "warn",
+        "semi": ["warn", "never"],
```

Comment:
The default setting in prettier, which we are using for webapp, is "Add a semicolon at the end of every statement": https://prettier.io/docs/en/options.html#semicolons

Reply:
Aligned